src/1-raw-data/mailinglists/mailman_data.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+//TODO
+// Add cycle to pull data by month if the start and end are multiple months
+// Run this monthly at start of new month to pull all new data
+
+import (
+	"log"
+	"strings"
+	"time"
+)
+
+func createMMURL(filename string, startDate string, endDate string) string {
+	url := *mailingListURL + "export/python-dev@python.org-" + filename + "?start=" + startDate + "&end=" + endDate
```

Comment:
You can do this, but you can also use fmt.Sprintf (https://golang.org/pkg/fmt/#Sprintf) which might be cleaner.

Reply:
Thanks for the insight here.